app/src/main/java/com/codesoom/assignment/controller/CatToyController.java

```diff
@@ -0,0 +1,87 @@
+package com.codesoom.assignment.controller;
+
+import com.codesoom.assignment.application.CatToyService;
+import com.codesoom.assignment.application.ToyService;
+import com.codesoom.assignment.domain.CatToy;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+
+/**
+ * 고양이 장난감에 대한 Http Request 요청을 처리합니다.
+ */
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/products")
+public class CatToyController {
+
+    private final ToyService<CatToy> catToyService;
+
+
+    /**
+     * 고양이 장남감 전체 목록을 조회한다.
+     * @return 장난감이 존재하지 않을 땐 빈 목록을 반환한다.
+     */
```

Comment:
- 리턴값을 설명하는 문구만 빼놓지 않아도 javadoc 주석은 평타를 칩니다. 
- "조회"하는지 아닌지에 대해서는 굳이 언급하지 않아도 충분해 보입니다.

```suggestion
    /**
     * 고양이 장난감 전체 목록을 리턴합니다.
     * @return 장난감 목록
     */
```